pyresample/geometry.py

```diff
@@ -195,6 +195,20 @@ def get_lonlats(self, data_slice=None, **kwargs):
         else:
             return self.lons[data_slice], self.lats[data_slice]
 
+    def get_lonlats_dask(self, chunks=CHUNK_SIZE):
```

Comment:
Why move this function ?

Reply:
It is used for SwathDefinitions, AreaDefinitions, and GridDefinitions. Previously it was not defined for GridDefinitions which are perfectly acceptable input area definitions.